README.md

```diff
@@ -3,66 +3,105 @@
+and higher-order machine learning and data science-informed insights. 
 
-```
+---
 
-Please use the build/run scripts in the /scripts folder to handle the
-construction of the Docker image and running the Docker container.
+## State of Development
 
-From the /8Knot folder, run:
+In the insightful words of Karl Fogel from his book "Producing Open Source Software":
 
-```bash
-bash scripts/build_docker_image.sh
+> "This is alpha software with known bugs. It runs, and works at least some of the time, but use at your own risk."
+
+We are incredibly happy to have people try our application in any state, and we doubly welcome any thorough bug reports.
+We would seriously recommend, however, that any conclusions drawn from this app, either realized from our deployed application or
+from a local instance, be scrutinized heavily until we make a proper, stable, >1.0 release.
+
+---
+
+## Communication
+
+Please feel free to join our [Matrix](https://matrix.to/#/#sandiego-rh:matrix.org) channel!
+
+We would prefer any initial communication be through Matrix but if you would prefer to talk to one of our maintainers, please feel free to peruse our AUTHORS.md file where you can find contact details.
+
+---
+
+## Usage Examples
+
+If you would like to see the current state of our application, we would love any user-stories or bug-reports from visiting our alpha-deployment!
+
+[8knot](https://eightknot.osci.io/)
+
+If you would prefer to look at our most up-to-date work, please check out the following section.
+
+---
+
+## Local Development
+
+We've tried to make it as easy as possible to go from fork/clone to a running instance.
+
+### Credentials
+
+You will need credentials of the following form, named "env.list", at the top-level of the 8knot directory that you clone.
+
+```
+    connection_string=sqlite:///:memory:
+    database=<<Database>>
+    host=<<Host>>
+    password=<<Your Password>>
+    port=<<Port>>
+    schema=augur_data
+    user=<<Your Username>>
+    user_type=read_only
 ```
 
-and then:
+To get these credentials please contact one of the maintainers of this project or ping us in our Matrix channel!
+
+### Runtime
+
+We use Docker containers to minimize the installation requirements for development. If you do not have Docker on your system, please follow the following guide: [Install Docker](https://docs.docker.com/engine/install/)
+
+### Scripts
+
+All runtime environment installation and architecture is handled in Docker containers.
+
+Anything not handled by Docker itself (building/networking/parameterizing containers) has
+been handled by the script "scripts/launch\_dev.sh". Please take a moment to become familiar with the steps in this script and suggest any improvements.
+
+Please run the following to launch your app:
 
 ```bash
-bash scripts/run_docker_container.sh
+bash scripts/launch_dev.sh
```

Comment:
Is this now functioning for the updated architecture?

Reply:
yep, sean merged the working from in the last PR I made